sway/config/output.c

```diff
@@ -482,23 +479,20 @@ static void queue_output_config(struct output_config *oc,
 		// same value as the clients'.
 		float adjusted_scale = round(scale * 120) / 120;
 		if (scale != adjusted_scale) {
-			sway_log(SWAY_INFO, "Adjusting output scale from %f to %f",
-				scale, adjusted_scale);
 			scale = adjusted_scale;
 		}
+		wlr_output_state_set_scale(pending, scale);
```

Comment:
nit: wlr_output_state_set_scale is invoked last for both branches. Either move it out or declare the `float scale;` inside the branches.

Reply:
Ah yeah that section is still rather silly. Cleaned it up quite a bit.